docs/output.md

```diff
@@ -2,44 +2,80 @@
 
 ## Introduction
 
-This document describes the output produced by the pipeline. Most of the plots are taken from the MultiQC report, which summarises results at the end of the pipeline.
+This document describes the output produced by the pipeline.
 
 The directories listed below will be created in the results directory after the pipeline has finished. All paths are relative to the top-level results directory.
 
-<!-- TODO nf-core: Write this documentation describing your workflow's output -->
-
 ## Pipeline overview
 
 The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes data using the following steps:
 
-- [FastQC](#fastqc) - Raw read QC
-- [MultiQC](#multiqc) - Aggregate report describing results and QC from the whole pipeline
+- [Summary tables folder](#summary-tables-folder) - Final tables that can be imported directly in R
```

Comment:
I think this part of the document is summarizing the steps and not the directory structure, right? In other words, this should go somewhere else I think.

Reply:
By looking at Ampliseq they have similar structure for tables as well
https://nf-co.re/ampliseq/2.7.0/docs/output

they call it "pipeline summary report"